cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesReports.java

```diff
@@ -194,4 +199,25 @@ public static void missingMandatoryAttributeReport(ReportNode reportNode, String
                 .withSeverity(TypedValue.ERROR_SEVERITY)
                 .add();
     }
+
+    public static ReportNode importingCgmesFileReport(ReportNode reportNode) {
+        return reportNode.newReportNode()
+                .withMessageTemplate("CGMESConversion", "Importing CGMES file(s)")
```

Comment:
A bit outside the move scope, but while we're at it could you add as values the extension and the basename in the template?

Reply:
in fact, I think that `getDataExtension()` will always be null for CGMES files, so don't add it to the report. Sorry for asking this! :slightly_frowning_face: